server/aws/README.ja.md

```
@@ -115,20 +115,30 @@ pnpm -F @trustknots/server-aws start:issuer
 Issuer metadata initialized
 Issuer is running on http://localhost:8081
 ```
 
+```text
+Authz server metadata initialized
+Authz is running on http://localhost:8082
+```
+
 2回目以降はデータが既に存在する場合スキップされます：
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)